students/minho/users/views.py

```diff
@@ -40,4 +40,23 @@ def post(self, request):
 			return JsonResponse({'MESSAGE' : 'SUCCESS'}, status = 201)
 			
 		except KeyError:
-			return JsonResponse({'MESSAGE' : 'KEY_ERROR'}, status = 400)
\ No newline at end of file
+			return JsonResponse({'MESSAGE' : 'KEY_ERROR'}, status = 400)
+
+class SignInView(View):
+	def get(self, request):
+		try :
+			data 	 = json.loads(request.body)
+
+			email    = data['email']
+			password = data['password']
+
+			if not User.objects.filter(email = email).exists():
+					return JsonResponse({'MESSAGE' : 'INVALID_USER'}, status = 401)
+			
+			if not User.objects.get(email = email).password == password:
+					return JsonResponse({'MESSAGE':'INVALIED_USER'}, status = 401)
+			
+			return JsonResponse({'MESSGAE' : 'SUCCESS'}, status = 200)
+
+		except KeyError :
+			return JsonResponse({'MESSAGE' : "KEY_ERROR"}, status = 400)
```

Comment:
민호님! indent 4칸으로 맞춰주세요~
```suggestion
class SignInView(View):
    def get(self, request):
        try :
	    data 	 = json.loads(request.body)
	    email    = data['email']
	    password = data['password']

	    if not User.objects.filter(email = email).exists():
		return JsonResponse({'MESSAGE' : 'INVALID_USER'}, status = 401)
	      
	    if not User.objects.get(email = email).password == password:
	        return JsonResponse({'MESSAGE':'INVALIED_USER'}, status = 401)
	      
	      return JsonResponse({'MESSGAE' : 'SUCCESS'}, status = 200)
      
      except KeyError :
	      return JsonResponse({'MESSAGE' : "KEY_ERROR"}, status = 400)
```

Reply:
수정했습니다!